src/main/scala/org/musigma/sbt/rat/SbtRatPlugin.scala

```diff
@@ -37,93 +39,163 @@ object SbtRatPlugin extends AutoPlugin {
   override def requires = JvmPlugin
 
   object autoImport {
-    // FIXME: this should probably be a task, not a configuration (e.g., packageBin)
-    val Audit = config("audit")
+    val ratCheck = taskKey[Unit]("Performs a release audit check")
```

Comment:
Thanks for handling this. I was originally following a pattern from sbt-site, but it didn't seem to be useful.